README.md

```diff
@@ -50,19 +51,19 @@ For a Maven project, add the following to your `pom.xml` file:
     <dependency>
       <groupId>io.dapr</groupId>
       <artifactId>dapr-sdk</artifactId>
-      <version>1.1.0</version>
```

Comment:
We have not released 1.2 yet. This is updated automatically after release is done. Remember that the SDK releases are not aligned to the runtime release.